libethash-cuda/ethash_cuda_miner_kernel.cu

```diff
@@ -26,7 +26,7 @@
 #endif
 
 __global__ void 
-__launch_bounds__(TPB, BPSM)
+//__launch_bounds__(TPB, BPSM)
```

Comment:
Generally, the fewer registers a kernel uses, the more threads and thread blocks are likely to reside on a multiprocessor, which can improve performance. Therefore, the compiler uses heuristics to minimize register usage while keeping register spilling and instruction count to a minimum. And this is achieved by using launch_bounds__(TPB, BPSM). 
It has 2 parameters:
**maxThreadsPerBlock**: specifies the maximum number of threads per block with which the application will ever launch the kernel
**minBlocksPerMultiprocessor**: is optional and specifies the desired minimum number of resident blocks per multiprocessor
However, for the ethminer code to run on GTX1060, the restriction of launch_bound is too tight (register number is limited to about 47, while the optimal should be about 70), which makes the overhead from register spilling to be higher than performance benefit from reducing register number. 
When deleting launch bound, we have about 2% performance improvement on GTX1060.